### DISCUSSION TITLE
How to disable Finder service | "New kitty tab here"

### DISCUSSION THREAD

**Discussion by jottr** [kovidgoyal/kitty#5258] [Q&A] [answered]
`kitty` adds itself to the right click menu in Finder.
How can I disable this feature so that it does not clutter my Finder menu?

<img width="263" alt="image" src="https://user-images.githubusercontent.com/2744198/178123841-f1dcfb7a-c10e-478a-abe2-e6ceae02a1be.png">

**Comment by page-down** [comment, marked as answer]
This is the same as other apps, go to System Preferences -> Keyboard -> Shortcuts -> Services, and select whatever you like.
